packages/colony-contract-loader-network/src/__tests__/NetworkLoader.test.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-env jest */
 
 import NetworkLoader from "../NetworkLoader";
+import EtherRouter from "../../contracts/static/EtherRouter.json";
```

Comment:
Does it makes sense to mock it here to be independent of the root?

Reply:
I suppose so, yes 🤔